stripe/src/main/java/com/stripe/android/view/CardInputWidget.java

```diff
@@ -514,9 +514,7 @@ public void onAnimationEnd(Animation animation) {
     @Override
     public void onWindowFocusChanged(boolean hasWindowFocus) {
         super.onWindowFocusChanged(hasWindowFocus);
-        if (hasWindowFocus) {
```

Comment:
Oops - no, that snuck in because I though I was going to have to do a lot of logic to figure out who had focus.